### PR TITLE
Fix comment history page

### PR DESCRIPTION
**Pull request by jfsoul**
## What does this change?
Safely accesses `config.page.keywordIds` to avoid error on comment history page for users in the brexit cohort.  Note: this is not the first time this has happened so we will look into if/how we can make the interface to config safer.

## What is the value of this and can you measure success?
Comment histories work again.

### CODE REVIEW

File: static/src/javascripts-legacy/projects/common/modules/experiments/tests/epic-for-brexit-cohort.js [hunk 1]
```diff
@@ -17,8 +17,8 @@ define([
 ) {
 
     function hasBrexitTag(){
-        var tags = config.page.keywordIds.concat(config.page.nonKeywordTagIds);
-        return tags.indexOf('politics/eu-referendum') > -1;
+        var tags = config.page.keywordIds && config.page.keywordIds.concat(config.page.nonKeywordTagIds);
```
Comment (gustavpursche) [inline static/src/javascripts-legacy/projects/common/modules/experiments/tests/epic-for-brexit-cohort.js]: There is also a `get()` method, which gives you safe access. Never used it, but it should be possible to do:

```js
config.get('page.keywordIds', []).concat()
```

**Review by lmath — APPROVED** [review]
:+1: I will remember this next time!

File: static/src/javascripts-legacy/projects/common/modules/experiments/tests/epic-for-brexit-cohort.js (hunk 1 above)
Comment (SiAdcock) [inline static/src/javascripts-legacy/projects/common/modules/experiments/tests/epic-for-brexit-cohort.js]: Ha! I just this morning discussed implementing this with @jfsoul. Completely missed the fact that it was already there ✅

Comment (gustavpursche) [inline static/src/javascripts-legacy/projects/common/modules/experiments/tests/epic-for-brexit-cohort.js]: Nice. Alex did this a while ago. :)

**Comment by jfsoul** [comment]
Thanks @gustavpursche, good shout!

**Pull request merged**